tests/test_dimerization_integration/TdT_kimmdy.yml

```
@@ -0,0 +1,32 @@
+dryrun: false
+name: 'TdT_RX'
+max_tasks: 100
+gromacs_alias: 'gmx'
+gmx_mdrun_flags: '-maxh 24 -dlb yes  -nt 64 -npme 0 -ntmpi 1' 
+ff: 'amber14sb_OL21_mod.ff' # optional, dir endinng with .ff by default 
+top: 'TdT_NPT.top'
+gro: 'TdT_NPT.gro'
+ndx: 'TdT_NPT.ndx'
+mds:
+  equilibrium_short:
+    mdp: 'TdT_md_short.mdp'
+  equilibrium_long:
+    mdp: 'TdT_md_long.mdp'
+  relax:
+    mdp: 'TdT_md_slow_growth.mdp'
+changer:
+  coordinates:
+    md: 'relax'  
+    slow_growth: 'morse_only'
+  topology:
+    parameterization: 'basic'
+reactions:
+  dimerization:
+    kmc: "rfkmc"
+
+sequence:
+- equilibrium_short
+- mult: 1
+  tasks:
+  - equilibrium_long
+  - reactions
```

Short run after reaction to make sure the system is ok.

```suggestion
  - reactions
  - equilibrium_short
```